index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/d31edfa6bf.js" crossorigin="anonymous"></script>
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+  <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+  <link rel="stylesheet" href="CSS/style.css">
+  <link rel="stylesheet" href="CSS/media.css">
+  <title>MMF 2022 | Mawazine Music Festival</title>
+  <link rel="icon" type="image/png" href ="/Assets/mawazine.png">
+</head>
+<body>
+
+  <header>
+    <div class="menu">
+      <div></div>
+    </div>
+    <nav class="first">
+      <ul>
+        <li><a href=""><i class="fa fa-twitter fa-x"></i></a></li>
+        <li><a href=""><i class="fa fa-facebook fa-x"></i></a></li>
+        <li> <select name="languages" id="lang">
+          <option value="English" selected>English</option>
+          <option value="Arabic">Arabic</option>
+          <option value="Spanish">Spanish</option>
+          <option value="French">French</option>
+        </select></li>
+        <li>My page</li>
+        <li>logout</li>
+      </ul>
+    </nav>
+    <div class="second">
+      <img class="logo" src="Assets/mawazine.png" alt="logo">
+      <nav class="nav-bar">
+        <ul>
+          <li><a href="index.html" class="nav-link" id="Home">Home</a></li>
+          <li><a href="about.html" class="nav-link">About</a></li>
+          <li><a href="#Program" class="nav-link">Program</a></li>
+          <li><a href="#Geusts" class="nav-link" >Guests</a></li>
+          <li><a class="nav-link" href="#Parteners">Partners</a></li>
+          <li><a href="#Envents" class="nav-link">Event</a></li>
+          <li><a href="#" class="nav-link"><span id="mawazine">mawazine</span> </a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+  <main>
```

Comment:
Consider matching the main section with the original design in terms of height, font styling, background, etc.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/182563260-4167bb6d-7257-4a76-95e4-77ba3dbd76e9.png) | ![image](https://user-images.githubusercontent.com/33268927/182563545-0eaa0c8b-b8ce-4c85-948f-a3044ef91c5d.png) |

Reply:
Consider matching the mobile version of this section to that of the original design. 
| Your design  | Original design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/182566748-f61dae84-4aa0-49ec-acc5-2aa71c483d67.png) | ![image](https://user-images.githubusercontent.com/33268927/182566853-881d3925-f07b-43d4-bb2d-31e83bb2873e.png) |

Reply:
- [ ] As the previous reviewer mentioned please make the header font transparent by adding [background-image](https://www.w3schools.com/cssref/pr_background-image.asp)